appveyor.yml

```diff
@@ -33,10 +33,10 @@ build:
   use_snupkg_format: true
   project: OSPSuite.Core.sln
 
-skip_branch_with_pr: true
+skip_branch_with_pr: false
 
 pull_requests: 
-  do_not_increment_build_number: true     
+  do_not_increment_build_number: false

```

Comment:
@abdelr @georgeDaskalakis  can we reset that to the default please?

Reply:
@msevestre Done